cate/ds/__init__.py

```diff
@@ -48,5 +48,8 @@ def cate_init():
     from .esa_cci_odp import set_default_data_store
     set_default_data_store()
```

Comment:
Only use for testing, e.g. set a flag `USE_ODP_LEGACY_DATA_STORE = True` or put in config